src/main/java/com/toofifty/easyblastfurnace/utils/MethodHandler.java

```diff
@@ -31,6 +31,12 @@ public void next()
         if (method == null) return;
         if (!state.getPlayer().isOnBlastFurnaceWorld()) return;
 
+        if (config.addCoalBuffer()) {
+            method.coalOffset = 0;
+        } else {
+            method.coalOffset = 1;
+        }
```

Comment:
Could this be retrieved JIT? I'm not sure if there's any point to storing the value. 
Maybe something like `state.getPlayer().getCoalOffset()`?

Reply:
I have replaced this with the method `state.getFurnace().getCoalOffset()`